snapcraft/tests/unit/commands/test_snap.py

```diff
@@ -140,12 +140,8 @@ def test_snap_containerized_remote(self,
         self.assertThat(result.exit_code, Equals(0))
 
         source = os.path.realpath(os.path.curdir)
-        self.assertIn(
-            "Using LXD remote 'myremote' from SNAPCRAFT_CONTAINER_BUILDS\n"
-            'Waiting for a network connection...\n'
-            'Network connection established\n'
-            'Mounting {} into container\n'.format(source),
-            fake_logger.output)
+        self.assertThat(fake_logger.output, Contains(
+            "Using LXD remote 'myremote' from SNAPCRAFT_CONTAINER_BUILDS"))
```

Comment:
Can you provide some context on what failed here? Checking the log messages has shown to be very valuable in the past to verify that things work correctly so I'd be wary of removing them (with no replacement).